library/liquibase

```diff
@@ -0,0 +1,12 @@
+Maintainers: Jake Newton <docker@liquibase.com> (@jnewton03)
+GitRepo: https://github.com/liquibase/docker.git
+
+Tags: latest, latest-alpine
+GitFetch: refs/heads/main
+GitCommit: 838c2b21806320fbb0118b21a34f140064301d10
+Directory: .
+
+Tags: 4.23, 4.23.1, 4.23-alpine, 4.23.1-alpine
+GitFetch: refs/heads/main
+GitCommit: 838c2b21806320fbb0118b21a34f140064301d10
+Directory: .
```

Comment:
All of these tags point to the same thing. It should only be one entry like

```
Tags: 4.23, 4.23.1, 4.23-alpine, 4.23.1-alpine latest, latest-alpine
GitFetch: refs/heads/main
GitCommit: 838c2b21806320fbb0118b21a34f140064301d10
Directory: .
```

Reply:
Done @LaurentGoderre and thank you for the review!